src/modules/rtpengine/rtpengine.c

```diff
@@ -2529,6 +2551,7 @@ static bencode_item_t *rtpp_function_call(bencode_buffer_t *bencbuf, struct sip_
 
 	body.s = NULL;
 	ng_flags.flags = bencode_list(bencbuf);
+	ng_flags.received-from = bencode_list(bencbuf);
```

Comment:
Typo here (dash instead of underscore). Maybe at least try to compile it before submitting?